handlers/telegram/telegram_test.go

```diff
@@ -693,7 +694,7 @@ var testCases = []ChannelHandleTestCase{
 		Data:                 errorFile,
 		ExpectedRespStatus:   200,
 		ExpectedBodyContains: "unable to resolve file",
-		ExpectedErrors:       []*courier.ChannelError{courier.ErrorResponseUnparseable("JSON")},
+		ExpectedErrors:       []*courier.ChannelError{courier.ErrorExternal("500", "error loading file")},
```

Comment:
this wasn't being checked because this we previously required a test case to return a message or status before we check channel logs